specification/batch/resource-manager/Microsoft.Batch/stable/2022-10-01/BatchManagement.json

```diff
@@ -5444,7 +5452,7 @@
             "description": "Setting up RBAC for Azure Batch VNets"
           }
         },
-        "dynamicVNetAssignmentScope": {
+        "dynamicVnetAssignmentScope": {
```

Comment:
This is to fix a S360 issue that was detected. Our api was returning dynamicVnetAssignmentsScope and not dynamicVNetAssignmentsScope. This was broken for a while

Reply:
reverted